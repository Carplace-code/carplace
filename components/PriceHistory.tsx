/* eslint-disable */

export default function PriceHistory({ history }: { history: any[] }) {
  return (
    <div className="flex h-64 items-center justify-center rounded-md border bg-white p-6">
      <p className="text-sm text-gray-500">
        {history.length === 0 ? "No price history available." : `[Price Chart Placeholder]`}
      </p>
    </div>
  );
}
