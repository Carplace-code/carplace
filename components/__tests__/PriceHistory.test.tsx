// components/__tests__/PriceHistory.test.tsx
import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";

import PriceHistory from "../PriceHistory";

describe("PriceHistory", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(<PriceHistory />);
    expect(container).toMatchSnapshot();
  });
});
