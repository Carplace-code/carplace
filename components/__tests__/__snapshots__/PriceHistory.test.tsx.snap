// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PriceHistory > renders correctly and matches snapshot 1`] = `
<div>
  <div
    class="flex h-64 items-center justify-center rounded-md border bg-white p-6"
  >
    <p
      class="text-sm text-gray-500"
    >
      [Price Chart Placeholder — last 60–90 days]
    </p>
  </div>
</div>
`;
